src/mem/DRAMInterface.py

```diff
@@ -1414,3 +1414,97 @@ class LPDDR5_6400_1x16_8B_BL32(LPDDR5_6400_1x16_BG_BL32):
     tCCD_L = "0ns"
     tRRD_L = "0ns"
     tWTR_L = "0ns"
+
+class LLM2(DRAMInterface):
+
+    beats_per_clock = 64
```

Comment:
Can you add a comment here that says "SERDES running at 32 GHz" assuming that is what `64` beats means.